ArduPlane/quadplane.cpp

```diff
@@ -538,6 +538,15 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("BCK_PIT_LIM", 38, QuadPlane, q_bck_pitch_lim, 10.0f),
 
+    // @Param: APPROACH_DIST
+    // @DisplayName: Q mode approach distance
+    // @Description: The minimum distance from the destination to use the fixed wing airbrake and approach code for landing approach. This is useful if you don't want the fixed wing approach logic to be used when you are close to the destination. Set to zero to use the default value of 1.5 times the calculated stopping_distance based on cruise airspeed and Q_TRANS_DECEL.
```

Comment:
```suggestion
    // @Description: The minimum distance from the destination to use the fixed wing airbrake and approach code for landing approach. This is useful if you don't want the fixed wing approach logic to be used when you are close to the destination. Set to zero to always use fixed wing approach. 
```
I don't see the behavior the descriptions described for the zero case. There is `transition_threshold` function, but that switches straight to air brake rather than pos 1.

Reply:
it does do a 1s airbrake, then goes to pos1. It skips the approach code

Reply:
I've changed it anyway